crates/oci-distribution/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "oci-distribution"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
can you revert this? We generally don't bump the version number until we're ready to cut a release.